core/modules/file/views/views_handler_field_file_uri.inc

```diff
@@ -43,12 +43,15 @@ class views_handler_field_file_uri extends views_handler_field_file {
     if (!empty($this->options['file_download_path']) && $data !== NULL && $data !== '') {
       $data = file_create_url($data);
       if (!empty($this->options['image_style'])) {
-      // $data contains url of image.
-      // Find insertion position of '/files'.
-        $insert_at = strpos($data, '/files');
-      // Text to insert in order to modify url.
+        // $data contains url of image.
+        // Get public file system path and its length.
+        $file_public_path = '/' . config_get('system.core','file_public_path');
```

Comment:
looks like we may be missing a space after this comma.

Reply:
Now added to PR.